### ISSUE TITLE
Fonts Library: Replace the Aa manage fonts icon with a button with visible text

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#58082]
### Description

Admittedly from anecdotal experience but every single time I use and test the font library I struggle to find the UI control to open the 'Manage fonts' modal dialog.

Clicking one of the installed fonrs does open the modal dialog but it brings me to the sub-view of that specific font within the modal dialot. This is useful when I want to manage _that_ specific font. 

Instead, when I want to open the Fonts library list, I struggle. 

Turns ouf the control to open it is a small `Aa` icon button. Screenshot:

![Aa icon button](https://github.com/WordPress/gutenberg/assets/1682452/02cfce1b-0dc0-4d31-9fee-48842de3cf36)

- The `Aa` icon doesn't really represent the concept of 'Manage fonts'.
- As a speech recognition software user I can't figure out the accessible name of that button by the visuals of the icon. As such, I wouldn't be able to issue an appropriate voice command and I would be forced to tab to the button to discover its accessible name. 
- The _same icon_ is used also for the Elements styles right below the Fonts group, which makes the UI confusing. What is the different meaning of the two icons?
- Icon buttons should only be used when there's not enough space to use visible text.
- Icons should be used when they help clarify meaning. They should not be used to obscure a functionality and make it less discoverable.

Overall, visually, looking at the UI I can't find anything that tells me 'Manage fonts'. I see icons, and I don't know what they mean nor the UI makes it easier to understand these icons meaning.

For accessibility, icon buttons are always less than ideal. In this case, I think this `Aa` icon does more harm than good and should be replaced with visible text. The placement of the button with visible text could be ~at the top~ [Edit:] at the bottom of the list of the installed fonts.

### Step-by-step reproduction instructions

- Go to the Site Editor > Styles > Typography
- Observe the UI within the Typography panel.
- Try to determine wich of the UI controls is the one to 'Manage fonts'.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
I'd ike to further clarify a design principle that seems obvious to me.

Besides the fact that controls that use only icons are inherently less than ideal for accessibility, Icons should help clarify the purpose of an UI control. They should help the users 'visual memory' to recognize an UI control immediately. As such, icons should be used unambiguously to always identify _one_ specific functionality.

IMHO, using the same icon for different features defeats the purpose of using icons in the first place and, to me, it's just an inappropriate way to use icons. Screenshot:

![Aa icon button](https://github.com/WordPress/gutenberg/assets/1682452/8ce8ad8b-edcd-4181-81ca-35176a5038c0)

**Comment by t-hamano** [comment]
There are two approaches I can think of, but what do you think?

Use the `cog` icon:

![image](https://github.com/WordPress/gutenberg/assets/54422211/86be4d57-b3a4-4ed7-aff8-b171aaa6afcb)

Use the link-like button:

![image](https://github.com/WordPress/gutenberg/assets/54422211/a1a7504a-b2d0-4262-b5a9-50e3e302e4d5)

**Comment by afercia** [comment]
> what do you think?

I'd like to do something entirely different.

Trying to look at the broader picture:

1
I'm not sure I like the proliferation of icons with different meaning placed at the end of a row, especially in the right panel. There's too many of them and they all have different meaning / purpose / functionality. Example screenshot:

![Screenshot 2024-01-23 at 09 54 03](https://github.com/WordPress/gutenberg/assets/1682452/981d1ca8-8da0-459e-b604-b8a070f4b59c)

Visually, and from a cognitive perspective, I think this is pretty overwhelming.

Some of these icons even change dynamically, for example the 'Dimensions' one, which may seem a smart idea at first but I'm not sure it helps clarity and predictability.

I think we should reduce the amount of different icons and functionalities used in that spot to a very small set of very recognizable patterns.

Keeping placing in that spot controls that do different things and are sometimes a little obscure because they use unclear, unique, icons is less than ideal to me.

2
I'm not sure there are other places in the editor UI where an icon button placed in that spot opens a modal dialog That is pretty a unique case and it's unexpected. Most of the icon buttons in that spot either open a dropdown menu or expand a panel or transform a control. I can't think of another icon button in that spot that opens a modal dialog.

3
The editor already uses a pattern for a 'manage all' button that opens a modal dialog. For example, the 'Explore all patterns' button. Screenshot:

![Screenshot 2024-01-23 at 09 39 23](https://github.com/WordPress/gutenberg/assets/1682452/69cf03a4-c315-4d9c-97ad-9209f8414642)

I think we should strive for consistency and establish a new pattern for all buttons that open a modal dialog. If it was up to me, I'd place a 'Manage all fonts' button at the end of the fonts list. Something like what is illustrated in the following example screenshot:

![Screenshot 2024-01-23 at 09 42 06](https://github.com/WordPress/gutenberg/assets/1682452/6473ee98-ac08-43bc-9749-dc8214678a95)

**Comment by t-hamano** [comment]
Thank you for the detailed explanation! I do think your approach makes sense.

> The editor already uses a pattern for a 'manage all' button that opens a modal dialog. For example, the 'Explore all patterns' button. Screenshot:

It may be a good idea to unify the "Edit something" and "Manage something" buttons that open modal dialogs using the secondary variation style.

**Comment by afercia** [comment]
Thanks for your feedback @t-hamano 

I'd say that the wording should be consistent as well. Currently:

'Explore all patterns'
'Manage all templates'
'Manage all template parts'

vs. 

'Manage fonts'

where the 'all' is missing. Either we always use 'all' or we don't.

**Comment by afercia** [comment]
Seems there is already feedback that the `Aa` icon button makes the Font Library very little discoverable. See 
https://github.com/WordPress/gutenberg/issues/55179

> I clicked the “Aa” icon to open up font management just because I followed your instructions, but I couldn’t have found the font management easily without them. I love icons, but this is only visible with a label.

Cc @annezazu

**Comment by annezazu** [comment]
Thanks for flagging. Going to tag in @WordPress/gutenberg-design to follow up here. I agree that we should improve discoverability but can't speak to design considerations.

**Comment by annezazu** [comment]
Ah! Apologies. I just saw this as soon as I wrote this of course: https://github.com/WordPress/gutenberg/pull/58158#issuecomment-1909557492 I think this is a solid approach.

**Comment by carolinan** [comment]
I was viewing a livestream by a WordPress contributor who is an educator and teaches people to use WordPress and even though I was guiding them they really struggled with finding the button.

**Comment by paaljoachim** [comment]
Switching to text will very much help. The Aa seems more decorative than it is actually a link. I have myself wondered about this icon earlier as it just did not seem like a link. Especially when as it is on the line with the FONTS heading. 

I really like the Manage all fonts blue outlined button which Andrea suggested above. It is big and it gives a very good hint to the user to click the button to manage all the fonts.

**Comment by annezazu** [comment]
Can we close this out in favor of https://github.com/WordPress/gutenberg/issues/58082 ? There are a few discussions happening, making it hard to keep track of feedback, designs, and attempted PRs.

**Comment by t-hamano** [comment]
> Can we close this out in favor of https://github.com/WordPress/gutenberg/issues/58082 ? There are a few discussions happening, making it hard to keep track of feedback, designs, and attempted PRs.

I also think that we should concentrate the discussion on one issue. Isn't the first reported issue #55179 appropriate?

**Comment by afercia** [comment]
No objections to closing this issue in favor of https://github.com/WordPress/gutenberg/issues/55179 as long as the feedback on this issue is reported there. To me, the part about _proliferation_ of icons pointed out in comment https://github.com/WordPress/gutenberg/issues/58082#issuecomment-1905648219 is important as well as the proposed soluton to use buttons with visible text.
I updated the labels on https://github.com/WordPress/gutenberg/issues/55179

**Comment by colorful-tones** [comment]
I agree that the icon as it exists is not ideal and located in an odd place.

After reading through #58082 and this issue (58082). I think @afercia is on to something ([comment above](https://github.com/WordPress/gutenberg/issues/58082#issuecomment-1905648219)):

> I think we should strive for consistency and establish a new pattern for all buttons that open a modal dialog. If it was up to me, I'd place a 'Manage all fonts' button at the end of the fonts list. Something like what is illustrated in the following example screenshot:

![image](https://github.com/WordPress/gutenberg/assets/405912/9592322d-e37a-4b4f-876b-975874f6deff)
